test/e2e_node/jenkins/conformance/conformance-jenkins.sh

```diff
@@ -21,11 +21,13 @@ set -e
 set -x
 
 : "${1:?Usage test/e2e_node/jenkins/conformance-node-jenkins.sh <path to properties>}"
-
+# TODO Verify that this is reasonable with Tim, I cannot find conformance-node-jenkins.sh but there is conformance-jenkins.sh
```

Comment:
I have no context on this.  Can someone from sig-testing weigh in?  @BenTheElder @cblecker @fejta ?

Reply:
I have no idea what these scripts were / are. FWIW there hasn't been a jenkins for quite some time now, but some things labeled "jenkins" are still in use somewhere..